locale/ko/LC_MESSAGES/index.po

```diff
@@ -142,131 +139,132 @@ msgid ""
+" Rufus가 번역되어 있다면 해당 번역자에게 감사의 말을 전해 주세요!"
 
 #: index.php:329
 msgid "Usage"
-msgstr "사용시 참고사항"
+msgstr "사용 방법"
 
 #: index.php:330
-msgid "Download the executable and run it &ndash; no installation is necessary."
-msgstr "다운로드 후에 실행합니다 &ndash; 프로그램을 설치할 필요가 없습니다."
+msgid ""
+"Download the executable and run it &ndash; no installation is necessary."
+msgstr "실행 파일을 다운로드하고 실행하기만 하면 됩니다. 프로그램을 설치할 필요가 없습니다."
 
 #: index.php:331
 msgid "The executable is digitally signed and the signature should state:"
-msgstr "실행파일은 아래와 같이 전자 서명이 되어 있습니다:"
+msgstr "실행 파일은 다음과 같이 전자 서명되어 있습니다:"
 
 #: index.php:332
 msgid "(v1.3.0 or later)"
-msgstr "(v1.3.0 혹은 그 이후 버전)"
+msgstr "(버전 1.3.0 이상)"
 
 #: index.php:333
 msgid "(v1.2.0 or earlier)"
-msgstr "(v1.2.0 혹은 그 이전 버전)"
+msgstr "(버전 1.2.0 이하)"
 
 #: index.php:334
 msgid "Notes on DOS support:"
-msgstr "DOS 지원을 위한 사항:"
+msgstr "DOS 지원 참고 사항:"
 
 #: index.php:335
 #, php-format
 msgid ""
 "If you create a DOS bootable drive and use a non-US keyboard, Rufus will "
-"attempt to select a keyboard layout according to the locale of your system. In "
+"attempt to select a keyboard layout according to the locale of your system."
+" In "
 "that case, <a target=\"_blank\" %s>FreeDOS</a>, which is the default "
 "selection, is recommended over MS-DOS, as it supports more keyboard layouts."
 msgstr ""
-"부팅가능한 DOS 드라이브할 때 영문 키보드를 사용하지 않으신다면, Rufus는 시스템"
-"에 맞는 지역의 키보드 레이아웃을 선택합니다. 이 경우에는 MS-DOS보다는 <a "
-"target=\"_blank\" %s>FreeDOS</a>를 권장하며, 기본값으로 되어 있습니다. FreeDOS"
-"는 더 많은 키보드 레이아웃을 제공하기 때문입니다."
+"DOS 부팅 USB 드라이브에서 미국 영어가 아닌 키보드를 사용해야 한다면, Rufus에서는 USB 드라이브를 만드는 시스템의 로캘에"
+" 따라서 키보드 레이아웃을 선택합니다. 미국 영어 이외의 키보드를 사용해야 하는 경우에는 MS-DOS 대신 더 많은 키보드 레이아웃을"
+" 지원하는 <a target=\"_blank\" %s>FreeDOS</a>(기본값) 사용을 권장합니다."
 
 #: index.php:336
 msgid "Notes on ISO Support:"
-msgstr "ISO 지원을 위한 사항:"
+msgstr "ISO 지원 참고 사항:"
 
 #: index.php:337
 #, php-format
 msgid ""
 "All versions of Rufus since v1.1.0 allow the creation of a bootable USB from "
 "an <a target=\"_blank\" %s>ISO image</a> (.iso)."
 msgstr ""
-"v1.1.0 이상 모든 버전의 Rufus는 <a target=\"_blank\" %s>ISO 이미지</a> (.iso)"
-"로 부터 부팅가능한 USB를 만드는 작업을 진행할 수 있습니다."
+"Rufus 버전 1.1.0 이상에서는 <a target=\"_blank\" %s>ISO 이미지</a>(.iso)로부터 부팅 가능한 USB"
+" 드라이브를 만들 수 있습니다."
 
 #: index.php:338
 #, php-format
 msgid ""
-"Creating an ISO image from a physical disc or from a set of files is very easy "
-"to do however, through the use of a CD burning application, such as the freely "
+"Creating an ISO image from a physical disc or from a set of files is very"
+" easy "
+"to do however, through the use of a CD burning application, such as the"
+" freely "
 "available <a target=\"_blank\" %s>CDBurnerXP</a> or <a target=\"_blank\" "
 "%s>ImgBurn</a>."
 msgstr ""
-"광학디스크내의 이미지 파일이나 ISO로 만들어지지 않은 파일들은 <a target="
-"\"_blank\" %s>CDBurnerXP</a> 혹은 <a target=\"_blank\" %s>ImgBurn</a> 과 같은 "
-"무료 소프트웨어를 이용하여  ISO 파일을 제작할 수 있습니다."
+"무료로 사용 가능한 <a target=\"_blank\" %s>CDBurnerXP</a>, <a target=\"_blank\" %s"
```

Comment:
On one side we used CD and on the other side we united what was called an "광 디스크":

">ImgBurn</a> 등 CD 작성 프로그램을 사용하여 광 디스크에서 ISO 이미지를 생성하거나 필요한 파일이 들어 있는 ISO 이미지를" -> ">ImgBurn</a> 등 CD 제작 프로그램을 사용해 CD에서 ISO 이미지를 생성하거나 필요한 파일이 들어 있는 ISO 이미지를"

Reply:
Technically it is possible to create ISO images from DVDs too and several Windows and Linux installation media are available as DVD too. Well, even the original string does not match in this regard, I would use "CD/DVD" instead of "CD".